.github/workflows/test_package.yml

```diff
@@ -9,16 +9,18 @@ jobs:
     name: unit-tests
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        python-version: [3.6]  #, 3.7, 3.8]
-        pytorch-version: ["1.8.0", "nightly"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
One version of Python only to save some compute.

Reply:
Okay, trying to figure out what versions actually work. Then I'll reduce the scope to one of those.

Reply:
3.12 seems to be the latest version that works, so I chose that.